content/about.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: page
+permalink: /about/
+title: About
+class: page-short anchors-center-fix
+---
+
+The CWL Project began out of a need for a standard to 
+describe analysis pipelines in a portable and reusable
+manner. Common Workflow Language (CWL) is a family of standards
+for the description of command line tools and of workflows made with these standards.
+CWL is designed to meet the needs of data-intensive science, such as Bioinformatics, Medical Imaging, Astronomy, High Energy Physics, and Machine Learning.
+
+
+The CWL project focus is on open standards, open-source code and an open community.
+
+
+<h2 class="cwl-red" id="vision">Vision of the CWL Project</h2>
+
+Researchers, scientists, and analysts share their batch data analysis workflows without technical barriers using an open standard. Sharing workflows this way is a usual occurrence and seen as a typical way of working. The workflows are complete and run on a variety of environments; and people re-use shared workflow descriptions and build new workflows from them. No vendor dominates the ecosystem
```

Comment:
```suggestion
Researchers, scientists, and analysts share their batch data analysis workflows without technical barriers using an open standard. Sharing workflows this way is a usual occurrence and seen as a typical way of working. The workflows are complete and run on a variety of environments; people can re-use shared workflow descriptions and build new workflows from them. No vendor dominates the ecosystem.
```

Reply:
Fine in principle, but this text was voted on by the CWL leadership team, so we'll have to get their approval later

Reply:
Should we merge this then?